regression-tests/24-compile-nrf52-ports/Makefile

```diff
@@ -11,7 +11,8 @@ hello-world/nrf52dk \
 nrf52dk/coap-demo/nrf52dk:coap-server \
 nrf52dk/coap-demo/nrf52dk:coap-client:SERVER_IPV6_ADDR=ffff \
 nrf52dk/mqtt-demo/nrf52dk \
-nrf52dk/blink-hello/nrf52dk
+nrf52dk/blink-hello/nrf52dk \
+nrf52dk/test-timer/nrf52dk
```

Comment:
Oops, sorry but I gave you a typo, is `timer-test`, that's probably why the test timeout